static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -39,6 +41,10 @@ define([
         setProgressTracker(atomId);
         tracking.track('play', getTrackingId(atomId));
 
+        if (closest(player.iframe, '.immersive-main-media__media')) {
+            document.querySelector('.immersive-main-media').classList.add('atom-playing');
+        }
+
```

Comment:
Ok cool, one last thing...closest(player.iframe, '.immersive-main-media__media') will return the same as document.querySelector('.immersive-main-media'). So to save making the query document.querySelector('.immersive-main-media') you can assign closest(player.iframe, '.immersive-main-media__media') to a variable, you can then use this variable in the if condition and also within the if condition's block.